cmake/common.cmake

```diff
@@ -342,6 +342,17 @@ if (PBFT)
   )
 endif()
 
+function(create_patched_enclave_lib name app_oe_conf_path enclave_sign_key_path)
+  set(patched_name ${name}.patched)
+  # configure_file(${CMAKE_CURRENT_BINARY_DIR}/lib${name}.so ${CMAKE_CURRENT_BINARY_DIR}/lib${patched_name}.so COPYONLY)
```

Comment:
Why is this commented out? Also, why are we doing that at build time?

Reply:
We are doing this at build time because the artifacts being copied for testing on the CI machines are only the signed libraries, and libloggingenc.so would not be copied. I wanted to avoid spamming the build artifacts (and the yml scripts) while also creating new artifacts during tests.
The comment has been removed.